packages/js-sdk/src/sandbox/signature.ts

```diff
@@ -12,36 +10,62 @@ import crypto from 'node:crypto'
  * @param expirationInSeconds Optional signature expiration time in seconds.
  */
 
-
 interface SignatureOpts {
-    path: string
-    operation: 'read' | 'write'
-    user: string
-    expirationInSeconds?: number
-    envdAccessToken?: string
+  path: string
+  operation: 'read' | 'write'
+  user: string
+  expirationInSeconds?: number
+  envdAccessToken?: string
+}
+
+// Cross-platform crypto implementation
+async function sha256(data: string): Promise<string> {
```

Comment:
Let's move this into some helper functions, this will be nice to re-use in the future